Library/Formula/docker-machine-driver-xhyve.rb

```diff
@@ -16,9 +16,13 @@ class DockerMachineDriverXhyve < Formula
 
   depends_on :macos => :yosemite
   depends_on "go" => :build
-  depends_on "docker-machine"
+  depends_on "docker-machine" => :recommended
 
   def install
+    unless build.with?("docker-machine")
+      opoo "You have disabled automatic installation of docker-machine; you are responsible for installing Machine yourself (e.g. via Docker Toolbox)."
+    end
```

Comment:
Please don’t print messages during the install. Plus most users will use the bottle and never see this message.